ArjunaJTA/jta/classes/com/arjuna/ats/internal/jta/transaction/arjunacore/jca/TransactionImporter.java

```diff
@@ -60,6 +61,8 @@
 	 */
     public SubordinateTransaction importTransaction(Xid xid, int timeout) throws XAException;
 
+    public TransactionImportResult importForeignTransaction(Xid xid, int timeout) throws XAException;
```

Comment:
I don't like the name "foreign" it implies its not a Narayana one. Maybe importRemoteTransaction?